src/Symfony/Component/Notifier/Bridge/Sweego/SweegoTransport.php

```diff
@@ -0,0 +1,91 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Notifier\Bridge\Sweego;
+
+use Symfony\Component\Notifier\Exception\TransportException;
+use Symfony\Component\Notifier\Exception\UnsupportedMessageTypeException;
+use Symfony\Component\Notifier\Message\MessageInterface;
+use Symfony\Component\Notifier\Message\SentMessage;
+use Symfony\Component\Notifier\Message\SmsMessage;
+use Symfony\Component\Notifier\Transport\AbstractTransport;
+use Symfony\Contracts\EventDispatcher\EventDispatcherInterface;
+use Symfony\Contracts\HttpClient\Exception\TransportExceptionInterface;
+use Symfony\Contracts\HttpClient\HttpClientInterface;
+
+/**
+ * @author Mathieu Santostefano <msantostefano@protonmail.com>
+ */
+final class SweegoTransport extends AbstractTransport
+{
+    protected const HOST = 'api.sweego.io';
+
+    public function __construct(
+        #[\SensitiveParameter] private string $apiKey,
+        ?HttpClientInterface $client = null,
+        ?EventDispatcherInterface $dispatcher = null,
+    ) {
+        parent::__construct($client, $dispatcher);
+    }
+
+    public function __toString(): string
+    {
+        return \sprintf('sweego://%s', $this->getEndpoint());
+    }
+
+    public function supports(MessageInterface $message): bool
+    {
+        return $message instanceof SmsMessage && null === $message->getOptions();
+    }
+
+    protected function doSend(MessageInterface $message): SentMessage
+    {
+        if (!$message instanceof SmsMessage) {
+            throw new UnsupportedMessageTypeException(__CLASS__, SmsMessage::class, $message);
+        }
+
+        $endpoint = \sprintf('https://%s/send', $this->getEndpoint());
+        $response = $this->client->request('POST', $endpoint, [
+            'headers' => [
+                'Api-Key' => $this->apiKey,
+            ],
+            'json' => [
+                'recipients' => [
+                    [
+                        'num' => $message->getPhone(),
+                        'region' => 'FR',
```

Comment:
@Wasta-Geek Is this `region` parameter mandatory? If yes, I guess I should make it configurable.

Reply:
Sorry for the delay
Yes this field is mandatory, it should be set according to the country code region of the phone number. This field consist of the ISO country code on two characters.

I guess the **campaign-type** should be also configurable; depending of the campaign-type, some special changes are sometimes applied to the message in order to respect laws depending of the country phone number; so it's in the user interest to fill the correct type according to it's uses